e2e_test/lib/forwards_to_isolate_async_worker.dart

```diff
@@ -21,7 +21,7 @@ class ForwardsToIsolateAsyncWorker extends AsyncWorkerLoop {
   ForwardsToIsolateAsyncWorker(this._isolateDriverConnection);
 
   @override
-  Future<WorkResponse> performRequest(WorkRequest request) {
+  Future<WorkResponse?> performRequest(WorkRequest request) {
```

Comment:
I would like to change the API here,  so that we always return a response.

I would propose that we handle this as the `DriverConnection` level, by returning a failed `WorkResponse` indicating that the connection was closed.

So the `MessageGrouper` classes would still return a nullable value from `next` but we would immediately handle that gracefully.

Reply:
Done, added `EXIT_CODE_BROKEN_PIPE = 32` to handle retries